ravenframework/ROMExternal.py

```diff
@@ -1,106 +0,0 @@
-# Copyright 2017 Battelle Energy Alliance, LLC
```

Comment:
Why is this file deleted?

Reply:
(And if it was deliberately deleted, what is the new file to use, since this is used by the HERON plugin?)

Reply:
@aalfonsi Can you answer this question?

Reply:
Hi @joshua-cogliati-inl,  probably it got mistakenly deleted when I merged multiple times devel (and fixed conflicts here and there) in the previous months. I will re-add it today or tomorrow when I address all the other comments.

Reply:
@joshua-cogliati-inl shouldn't this cause a failure of a test in RAVEN? or it is not tested yet?

Reply:
Um, in this pull request, you removed the test from tests/framework/ROM/pickleTests/tests ;)

Reply:
ohhhhh O_O :( :( sorry

Reply:
I re-added the test for this, but not the ROMExternal.py file. I would like you to check the entire commit for anything mistakenly deleted or modified and fix them.

Reply:
readied